internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/controller/TestClusterLevelConfigForActiveActiveReplication.java

```diff
@@ -72,6 +74,12 @@ public void testClusterLevelActiveActiveReplicationConfigForNewHybridStores() th
     // Check store level Active/Active is enabled or not
     assertFalse(store.isActiveActiveReplicationEnabled());
 
+    waitForNonDeterministicPushCompletion(
+        Version.composeKafkaTopic(storeName, 1),
+        parentControllerClient,
```

Comment:
Why are we adding this? We didn't create topic in parent Kafka cluster in the test before?

Reply:
These are actually failing because we were updating config while push was running.

Reply:
Then why adding a wait time here could help?

Reply:
Wait will make sure that the push has finished, then we can convert it to hybrid.

Reply:
I see! Thanks for clarification.